students/Jaeyou-Jeong/westagram/users/views.py

```diff
@@ -0,0 +1,32 @@
+import json
+
+from django.http import JsonResponse
+from django.views import View
+
+from users.models import Signin
+from users.checks import email_validation
+
+class Sign(View):
+    def post(self, request):
+        data = json.loads(request.body)
+        try:
+            if len(data['password']) < 8:
+                return JsonResponse({"message": "비밀번호는 8자리 이상으로 만들어 주세요."}, status=400)
```

Comment:
리턴 메시지는 클라이언트에게 보내는 것이 아니고 프론트에게 보내는 것입니다. 굳이 한글로 친절하게 알려주시지 않으셔도 됩니다.
```suggestion
                return JsonResponse({"message": "INVALID_PASSWORD"}, status=400)
```

Reply:
네 수정했습니다.